lib/Sema/TypeCheckConcurrency.cpp

```diff
@@ -512,6 +512,11 @@ static bool varIsSafeAcrossActors(const ModuleDecl *fromModule,
     if (var->getAttrs().hasAttribute<NonisolatedAttr>())
       return true;
 
+    // Static 'let's are initialized upon first access, so they cannot be
+    // synchronously accessed across actors.
+    if (var->isGlobalStorage() && var->isLazilyInitializedGlobal())
```

Comment:
I think this rule prohibits reasonable code from working and would like to discuss it further before merging.

Reply:
@hborla has clarified that I misread the code, and this is already appropriate narrow in applicability.